docs/how-to/contribute.rst

```diff
@@ -4,7 +4,7 @@
 How to contribute to Toga
 =========================
 
-If you experience problems with Toga, `log them on GitHub`_. If you want to contribute code, please `fork the code`_ and `submit a pull request`_.
+If you experience problems with Toga, `log them on GitHub`_. If you want to contribute code, please `for the code`_ and `submit a pull request`_.
```

Comment:
I feel like this would read better as

```
If you experience any problems with Toga, create an issue on GitHub. If you would like to contribute to Toga, please fork the code and submit a pull request.
```